app/Controllers/App.php

```diff
@@ -2,8 +2,7 @@
 
 namespace App\Controllers;
 
-use BCcampus\OpenTextBooks\Controllers\Catalogue;
-use BCcampus\OpenTextBooks\Controllers\Reviews;
```

Comment:
Removing these creates a fatal error on line 130 in the same file. Please revert.